### ISSUE TITLE
Failed to load symbolec_init_tables

### ISSUE THREAD

**Issue by vndroid** [intel/isa-l#264]
Debian 11 build it for Hadoop, as:

```bash
git clone https://github.com/intel/isa-l
./autogen.sh
./configure --prefix=/usr --libdir=/usr/lib
make
sudo make install
```

Then  build the Hadoop, The error is:

```bash
Native library checking:
hadoop:  true /opt/hadoop-3.3.6/lib/native/libhadoop.so.1.0.0
zlib:    true /lib/aarch64-linux-gnu/libz.so.1
zstd  :  true /lib/aarch64-linux-gnu/libzstd.so.1
bzip2:   true /lib/aarch64-linux-gnu/libbz2.so.1
openssl: true /lib/aarch64-linux-gnu/libcrypto.so.1.1
ISA-L:   false Loading ISA-L failed: Loading functions from ISA-L failed: Failed to load symbolec_init_tables
PMDK:    true /usr/lib/aarch64-linux-gnu/libpmem.so.1.0.0
```

Env:

OS: Debian 11 (Oracle Cloud)
Arch: aarch64
CPU: Neoverse-N1

Why is this? How to resolve it?

**Comment by pablodelara** [comment]
Hi @vndroid. This PR should fix the issue https://github.com/intel/isa-l/pull/259. Can you check?

**Comment by vndroid** [comment]
> Hi @vndroid. This PR should fix the issue #259. Can you check?

Okay, I will try it later.

**Comment by vndroid** [comment]
> Hi @vndroid. This PR should fix the issue #259. Can you check?

works well on arrch64! Thx.

```bash
Native library checking:
hadoop:  true /opt/hadoop-3.3.6/lib/native/libhadoop.so.1.0.0
zlib:    true /lib/aarch64-linux-gnu/libz.so.1
zstd  :  true /lib/aarch64-linux-gnu/libzstd.so.1
bzip2:   true /lib/aarch64-linux-gnu/libbz2.so.1
openssl: true /lib/aarch64-linux-gnu/libcrypto.so
ISA-L:   true /lib/libisal.so.2
PMDK:    true /usr/lib/aarch64-linux-gnu/libpmem.so.1.0.0
```

No Error report.
